nunaserver/docker-compose.yml

```diff
@@ -24,6 +24,8 @@ services:
       - /tmp:/tmp
     network_mode: host
     environment:
+        # change this
+      - CSRF_SECRET_KEY=abc123
```

Comment:
Does this change need to go into `deployment/docker-compose.prod.yml` too?  If so, will need to modify GHA to set the password on deployment.

Reply:
Yep sorry did not notice it there, I'll add the code in a few hours to docker-compose.prod and .dev and take a look at adding to ci

Reply:
Yeah reading through the github actions, I can add the csrf sed, but the token itself need to be added by a maintainer. A random 32char string should be perfect for it

Reply:
Yes, I have added a new secret called CSRF_SECRET_KEY.  Could add the following line to the `production.yml` GHA.

`sed -r -i 's/(CSRF_SECRET_KEY=).*$/\1${{ secrets.CSRF_SECRET_KEY }}/g' docker-compose.yml`

Reply:
yep its added with the latest commit